src/main/java/org/sunbird/workflow/service/UserBulkUploadService.java

```diff
@@ -272,36 +273,50 @@ private void processBulkUpload(HashMap<String, String> inputDataMap) throws IOEx
+                    employmentDetailsKey.add(Constants.CADRE);
+                    employmentDetailsKey.add(Constants.PAY_TYPE);
                     WfRequest wfRequest = this.getWFRequest(valuesToBeUpdate, userId);
                     List<HashMap<String, Object>> updatedValues = new ArrayList<>();
                     for(Map.Entry<String, String> entry : valuesToBeUpdate.entrySet()){
                         String fieldKey;
                         HashMap<String, Object> updatedValueMap = new HashMap<>();
                         updatedValueMap.put(entry.getKey(), entry.getValue());
                         HashMap<String, Object> updateValues = new HashMap<>();
-                        updateValues.put("fromValue", new HashMap<>());
-                        updateValues.put("toValue", updatedValueMap);
+                        updateValues.put(Constants.FROM_VALUE, new HashMap<>());
+                        updateValues.put(Constants.TO_VALUE, updatedValueMap);
                         if(employmentDetailsKey.contains(entry.getKey())){
-                            fieldKey = "employmentDetails";
+                            fieldKey = Constants.EMPLOYMENT_DETAILS;
                         } else{
-                            fieldKey = "professionalDetails";
-                        }
-                        updateValues.put("fieldKey", fieldKey);
+                            fieldKey = Constants.PROFESSIONAL_DETAILS;
+                         }
+                        updateValues.put(Constants.FIELD_KEY, fieldKey);
                         updatedValues.add(updateValues);
                         if(null != wfRequest){
                             wfRequest.setUpdateFieldValues(updatedValues);
                         }
                         userProfileWfService.updateUserProfileForBulkUpload(wfRequest);
                         WfStatusEntity wfStatusEntityFailed = wfStatusRepo.findByWfId(wfRequest.getWfId());
                         if(null != wfStatusEntityFailed && Constants.REJECTED.equalsIgnoreCase(wfStatusEntityFailed.getCurrentStatus())){
-                            this.setErrorDetails(str, Collections.singletonList("UPDATE FAILED"), statusCell, errorDetails);
+                            this.setErrorDetails(str, Collections.singletonList(Constants.UPDATE_FAILED), statusCell, errorDetails);
                             failedRecordsCount++;
                         } else{
-                            statusCell.setCellValue(Constants.SUCCESS_UPPERCASE);
+                            if(userRecordUpdate){
+                                noOfSuccessfulRecords++;
+                                statusCell.setCellValue(Constants.SUCCESS_UPPERCASE);
+                            } else{
+                                this.setErrorDetails(str, Collections.singletonList(Constants.UPDATE_FAILED), statusCell, errorDetails);
+                                failedRecordsCount++;
+                            }
```

Comment:
userRecordUpdate -- this is not updated here
do we need this condition ?

Reply:
This is required at the end to updates the status column for each row in the excel sheet.